internal/pkg/cli/svc_package.go

```diff
@@ -150,6 +160,34 @@ func newWorkloadStackGenerator(o *packageSvcOpts) (workloadStackGenerator, error
 	return deployer, nil
 }
 
+func newTemplateOverrider(pathToOverridesDir, app, env string, fs afero.Fs, provider *sessions.Provider) (clideploy.Overrider, error) {
+	info, err := override.Lookup(pathToOverridesDir, fs)
```

Comment:
Do we want to add a unit test for this?

Reply:
Done! I've moved the constructor to the `deploy/cli` pkg and added tests